Extending/Backoffice-Search/index.md

```diff
@@ -0,0 +1,84 @@
+---
+keywords: Backoffice Search
+versionFrom: 8.6.0
+meta.Title: "Backoffice Search"
+meta.Description: "A guide to customization of Backoffice Search"
+---
+
+# Backoffice Search
+
+Backoffice search is usefull tool, which support you in finding your content. Umbraco by default will allow you to search using only few fields: 
+
+| Node Type    | Propagated Fields    |
+| ------------ | -------------------- |
+| All Nodes    | UmbracoFileFieldName |
+| Media Nodes  | File path            |
+| Member Nodes | email, loginName     |
+|              |                      |
+
+Umbraco 8.6 introduce new easy way for extend/override that fields, which is called IUmbracoTreeSearcherFields. This service exposed hardcoded internal list of fields and allow for modification on them.
```

Comment:
```suggestion

However, a specific Umbraco implementation may have additional custom properties that it would be useful to be considered in a Backoffice Search, for example perhaps there is an 'Organisation Name' property on the Member Type, or the 'Main Body Text' property of a Content item. 

Umbraco 8.6 introduced a new easy way to extend/override the default fields by implementing `IUmbracoTreeSearcherFields`. This service exposes the list of default internal search fields for each section and allows modification of them.
```